platform/view/view/session.go

```diff
@@ -53,9 +55,15 @@ type Session interface {
 	// Send sends the payload to the endpoint
 	Send(payload []byte) error
 
+	// SendWithContext sends the payload to the endpoint with the passed context
```

Comment:
does this mean that the context is part of the message or the context is used to control the send operation itself?

Reply:
no it's part of the message. it carries some information. in this case the span context

Reply:
I would clarify in the comment then

Reply:
No but the context doesn't have to do only with the span. The span uses the context but the context has more functions in general, e.g. canceling the context, or adding baggage values